main.nf

```diff
@@ -2544,6 +2607,7 @@ process get_software_versions {
     sexdeterrmine --version &> v_sexdeterrmine.txt || true
     kraken2 --version | head -n 1 &> v_kraken.txt || true
     endorS.py --version &> v_endorSpy.txt || true
+    pileupCaller --version &> v_sequencetools.txt 2>&1 || true
```

Comment:
You need to add the (python) regex for passing the version number here into the `bin/scrapr_software_versions.py` file